packages/ckeditor5-dev-env/lib/release-tools/utils/transformcommit.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const getPackageJson = require( './getpackagejson' );
+
+const transformCommit = {
```

Comment:
1. The file name `transformCommit` is suggesting, that the default export should be a function, which is not. 
2. This file name should contains `-` between words according to our style guide :D

Maybe `transform-commit-utils`/ `transform-commit-helpers`?